### PR TITLE
TOMEE-2813 - Translate to Portuguese: examples/dynamic-proxy-to-access-mbean	

### PR DESCRIPTION
**Pull request by Daniel-Dos**
Translate to portuguese of example/dynamic-proxy-to-access-mbean

link jira: https://issues.apache.org/jira/browse/TOMEE-2813

### CODE REVIEW

**Comment by Daniel-Dos** [comment]
thanks :  )

File: examples/dynamic-proxy-to-access-mbean/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+:jbake-type: page
+:jbake-status: status=published
+= Proxy dinâmico para acessar o MBean
+
+== Exemplo
+
+Acessar o MBean é algo simples por meio da API JMX, mas geralmente é
+técnico e não muito interessante.
+
+Este Exemplo simplifica este trabalho simplesmente fazendo isso genericamente em um proxy.
```
Comment (hbelmiro) [inline examples/dynamic-proxy-to-access-mbean/README_pt.adoc:11]:
```suggestion
Este exemplo simplifica este trabalho simplesmente fazendo isso genericamente em um proxy.
```

Comment (hbelmiro) [inline examples/dynamic-proxy-to-access-mbean/README_pt.adoc:11]:
```suggestion
Este exemplo simplifica este trabalho simplesmente fazendo isso genericamente em um proxy.
```

File: examples/dynamic-proxy-to-access-mbean/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+
+Portanto, do lado do usuário, você simplesmente declara uma interface para acessar seu
```
Comment (hbelmiro) [inline examples/dynamic-proxy-to-access-mbean/README_pt.adoc:13]:
```suggestion
Portanto, do lado do usuário, você simplesmente declara uma interface para acessar seus
```

File: examples/dynamic-proxy-to-access-mbean/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+MBeans.
+
+Nota: A implementação de exemplo usa um MBeanServer local, mas aprimora
```
Comment (hbelmiro) [inline examples/dynamic-proxy-to-access-mbean/README_pt.adoc:16]:
```suggestion
Nota: A implementação de exemplo usa um MBeanServer local, mas aprimorando
```

File: examples/dynamic-proxy-to-access-mbean/README_pt.adoc [hunk 4]
```diff
@@ -0,0 +1,451 @@
...
+aa API de exemplo, é fácil imaginar uma conexão remota com
```
Comment (hbelmiro) [inline examples/dynamic-proxy-to-access-mbean/README_pt.adoc:17]:
```suggestion
a API de exemplo, é fácil imaginar uma conexão remota com
```

File: examples/dynamic-proxy-to-access-mbean/README_pt.adoc (hunk 1 above)
Comment (Daniel-Dos) [reply examples/dynamic-proxy-to-access-mbean/README_pt.adoc]: done thanks

File: examples/dynamic-proxy-to-access-mbean/README_pt.adoc (hunk 3 above)
Comment (Daniel-Dos) [reply examples/dynamic-proxy-to-access-mbean/README_pt.adoc]: done thanks

File: examples/dynamic-proxy-to-access-mbean/README_pt.adoc (hunk 4 above)
Comment (Daniel-Dos) [reply examples/dynamic-proxy-to-access-mbean/README_pt.adoc]: done thanks

**Comment by Daniel-Dos** [comment]
thanks @hbelmiro  for review : )

**Pull request merged**